lib/base-compiler.js

```diff
@@ -509,7 +509,7 @@ class BaseCompiler {
 
             if (fs.existsSync(passDump) && fs.statSync(passDump).isFile()) {
                 output.currentPassOutput = fs.readFileSync(passDump, 'utf-8');
-                if (output.currentPassOutput.match('^\s*$')) {
+                if (output.currentPassOutput.match('^s*$')) {
```

Comment:
Hmm. I think this is meant to be matching whitespace; not an 's' character. I think this should be `^\\s*$'` instead or `/^\s$/`